html/elements/source.json

```
@@ -37,6 +37,47 @@
             "deprecated": false
           }
         },
+        "height": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "90"
+              },
+              "chrome_android": "mirror",
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "106",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.picture_source_dimension_attributes.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
```

HI Dave, 
We now know this is shipping in 108 too so I'm going to merge updates into your PR.

```suggestion
              "firefox": [
                {
                  "version_added": "108"
                },
                {
                  "version_added": "106",
                  "version_removed": "108",
                  "flags": [
                    {
                      "type": "preference",
                      "name": "dom.picture_source_dimension_attributes.enabled",
                      "value_to_set": "true"
                    }
                  ]
                }
              ],
```

---

@hamishwillee thanks for this and it also makes things much clearer too.